### PR TITLE
FH-3617 - Add FHC tool project into http://feedhenry.org/projects/

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3617

Following the result (local test):

screen shot 2018-01-12 at 11 35 04 am

### CODE REVIEW

**Comment by camilamacedo86** [comment]
Hi @damienomurchu, @pmdarrow could give a help here? Could you check this one?

**Review by damienomurchu — COMMENTED** [review]
Changes look good. Having a few ruby issues testing it locally, but will test once I can build

**Comment by damienomurchu** [comment]
cc'ing @secondsun for review also

File: projects.md [hunk 1]
```diff
@@ -78,6 +78,20 @@ send push notifications to any device, regardless of platform or network.
 
 <div class="project-title" markdown="1">
 
+## FHC
+
+FeedHenry Command Line Interface
```
Comment (damienomurchu) [inline projects.md:83]: @camilamacedo86 I think it's fine to include a new category and the FHC repo under it. I think the category should be more general though - perhaps `FH Tooling`, so other repos other than FHC can be included in it and not just have a category with one repo within it

/my2cents

cc'ing @david-martin for his thoughts

Comment (david-martin) [reply projects.md:83]: If there are other tools, sure.
If its just FHC, i don't think its that much a problem. But maybe the category could be 'CLI' then

Comment (camilamacedo86) [reply projects.md:83]: @damienomurchu @david-martin  I don't know any other project that we can add here in order to create a category CLI or FH Tooling. However, please let me know if you would like to change it anyway.

**Comment by camilamacedo86** [comment]
@damienomurchu @david-martin I don't know any other project that we can add here in order to create a category CLI or FH Tooling. However, please let me know if you would like to change it anyway. What is required here for we move forward?

**Comment by damienomurchu** [comment]
@camilamacedo86  I'm not sure who the maintainer of feedhenry.org is these days (or if there is one), but changes look fine to me (sorry for late reply, was OOO all last week). 

There were some plans last year to house more content on feedhenry.org (like upstream docs), but not sure how those plans are now, and I really only mention it in case there were any grander active plans for feedhenry.org so that any changes fitted in with them. 

:+1: for me to merge these changes in, so this PR doesn't linger though :)

cc @secondsun @finp

**Review by damienomurchu — APPROVED** [review]
Code looks good, and built changes locally also, all good to me :+1:

**Comment by secondsun** [comment]
I think a lot of things are moving to aerogear.org, I'll check out your stuff and probably merge it today.

**Comment by camilamacedo86** [comment]
@secondsun @damienomurchu Tks for the information provided.  I don't have permission in this repo to do the merge :-( Please, could you do it?

**Comment by camilamacedo86** [comment]
@davidffrench could you please do the merge here? Ps. I don't have permission to do it.

**Comment by davidffrench** [comment]
@camilamacedo86 I have the ability to merge this. However, i am not familiar with the process of getting this changed pushed to feedhenry.org. @secondsun are you familiar with this process?

**Comment by camilamacedo86** [comment]
@secondsun , damienomurchu could you please merge it?

**Comment by damienomurchu** [comment]
Reviewed the changes, built the code & all seems fine. If I'm overlooking something, changes can be reverted (& I am happy to help with a corrective PR if so), but otherwise am merging in these changes now

**Pull request merged**